src/appUrls/index.js

```diff
@@ -0,0 +1,10 @@
+// const BASE_URL = process.env.API_URL;
+export const BASE_URL = "https://ah-backend-lannister-staging.herokuapp.com";
+// export const AUTH_TOKEN = "auth_token";
+export const AUTH_TOKEN =
+  "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6ImFkbWluQGFkbWluLmNvbSIsImV4cCI6MTU0MTYwMDgyNSwidXNlcm5hbWUiOiJhZG1pbiJ9._5fX1AE0RoyvSnx7AJbndt9VWhNH1iM96Rc6JIlx_Zo";
+
```

Comment:
Replace the auth-token value with an environment variable so as not to expose such sensitive data

Reply:
This token had been added to the environment variable. Though it's a dummy token that i keep changing so as to mimic an authenticated user, it expires after some time. But I plan to rebase with social auth login so that I can get an authenticated user

Reply:
Have u rebased with this branch?

Reply:
Yes i have rebased with develop